lib/godwoken_explorer/udt.ex

```diff
@@ -75,9 +77,68 @@ defmodule GodwokenExplorer.UDT do
     end
   end
 
-  def count_holder(udt_id) do
-    from(cbub in CurrentBridgedUDTBalance, where: cbub.udt_id == ^udt_id)
-    |> Repo.aggregate(:count)
+  def count_holder(udt) do
+    case udt.type do
+      :native ->
+        bridge_udt_query = from(u in UDT, where: u.bridge_account_id == ^udt.id)
+
+        case Repo.one(bridge_udt_query) do
+          nil ->
+            from(cub in CurrentUDTBalance,
+              where: cub.token_contract_address_hash == ^udt.contract_address_hash
```

Comment:
here need to filter value is 0 records

Reply:
fix here: https://github.com/Magickbase/godwoken_explorer/pull/849/files#diff-d44d00acb06eaf71b14ba5b0fb3233b779927732d67685af828afda5e67694af